README.md

```diff
@@ -103,8 +103,17 @@ pasta does not seem to work well
 Use scripts in [`./init-host`](./init-host) for automating these steps.
 
 ## Usage
+
 See `make help`.
 
+If you are running a multi-node setup with a shared filesystem and location for your network CNI files, you will want to create a non-shared location for each node's usernetes code (e.g., `/tmp` is usually not shared) and run this additional command for each of the control-plane and worker nodes before `make up`. It will give the network (and corresponding CNI files) unique names in the shared location, usually in `~/.config/cni`,
```

Comment:
I don't think that container engines support locating CNI files on a shared filesystem

Reply:
I suggest just mounting a local filesystem on .config/cni

Reply:
> I suggest just mounting a local filesystem on .config/cni

You mean on the HPC node? That seems overkill for what comes down to a file naming issue.

The solution here does not change functionality for a user that doesn't need this change, but supports multi-node shared filesystem setups for users that need it with an isolated `make multi-node` command. If there turns out to be other multi-node functionality that is needed, it could be added to that section.

Reply:
> I don't think that container engines support locating CNI files on a shared filesystem

In rootless mode, podman puts the cni files in the user's home. To be clear, it isn't shared between users, it is shared between nodes. [reference](https://docs.podman.io/en/stable/markdown/podman.1.html),

Reply:
Same as:
- https://github.com/rootless-containers/usernetes/pull/374#discussion_r2168405158

I don't think a new Makefile target should be added for this.
`docker-compose.yaml` can be modified in `vi` or `yq`.

Reply:
They are expected/guaranteed to be under the home, but not expected to be under the shared home

Reply:
> They are expected/guaranteed to be under the home, but not expected to be under the shared home

I have never seen an HPC cluster with a user home that is not a filesystem mapped across nodes, and thus shared. It's usually NFS. It's strategically like that so you can login to multiple different clusters an see files, and jobs running across compute nodes can see the same space too.